src/main/resources/de/unijena/cheminf/mortar/message/Message_en_GB.properties

```diff
@@ -544,4 +544,24 @@ IMoleculeFragmenter.CycleFinderOption.relevant.tooltip = Relevant cycles of a gr
 IMoleculeFragmenter.CycleFinderOption.tripletShort.displayName = triplet short
 IMoleculeFragmenter.CycleFinderOption.tripletShort.tooltip =  The shortest cycle through each triple of vertices; allows to generate the envelope rings of some molecules (e.g. naphthalene) without generating all cycles
 IMoleculeFragmenter.CycleFinderOption.vertexShort.displayName = vertex short
-IMoleculeFragmenter.CycleFinderOption.vertexShort.tooltip = The shortest cycles through each vertex; linear independence is not checked; in practice similar to MCB
\ No newline at end of file
+IMoleculeFragmenter.CycleFinderOption.vertexShort.tooltip = The shortest cycles through each vertex; linear independence is not checked; in practice similar to MCB
+##AlkylStructureFragmenter##
+AlkylStructureFragmenter.displayName = Alkyl Fragmenter
+AlkylStructureFragmenter.fragmentSaturationSetting.tooltip = Defines how open valences resulting from bond breakages during fragmentation should be saturated
+AlkylStructureFragmenter.fragmentSaturationSetting.displayName = Fragment saturation setting
+AlkylStructureFragmenter.keepNonFragmentableMoleculesSetting.tooltip = If set to true, enables non-fragmentable molecules to stay in the fragmenter pipeline and be returned together with all received fragments.
+AlkylStructureFragmenter.keepNonFragmentableMoleculesSetting.displayName = Keep non-fragmentable molecules in Pipeline
+AlkylStructureFragmenter.maxChainLengthSetting.tooltip = Defines maximum length of chains: 0 is no restrictions, 1 returns methanes
```

Comment:
Defines maximum length of the alkyl chain fragments generated; 0 leads to no fragmentation, 1 produces only methane fragments, etc.